ntbs-service/wwwroot/css/notification.scss

```diff
@@ -25,3 +25,8 @@
 .notifications-table {
     margin-top: 0;
 }
+
+.delete-notification-button {

```

Comment:
Needed to make the buttons appear one on top of the other instead of side by side

Reply:
We could probably make this slightly more generic and apply it to the other buttons in that area? 
Hypothetically if that section gets more width it sounds like currently it will start behaving strangely right?

Reply:
yes, definitely worth applying same treatment across the actions area